packages/cli/src/util/input/list.ts

```diff
@@ -54,7 +54,7 @@ export default async function list(
     eraseFinalAnswer = false, // If true, the line with the final answer that inquirer prints will be erased before returning
   }: ListOptions
 ): Promise<string> {
-  require('./patch-inquirer-legacy');
+  require('./patch-inquirer');
```

Comment:
I changed patch-inquirer-legacy to patch-inquirer to override chalk that inquirer list directly use.

If there is any concern please leave a comment.

Reply:
Ideally this would be a separate PR to keep the diff noise down in this one.

Reply:
I separated PR https://github.com/vercel/vercel/pull/8919